src/templates/expressions/fields.ts

```diff
@@ -2,14 +2,15 @@ import {
     TsExpression,
     toTsExpressions,
 } from "./expression";
+import { KintoneFieldTypeToTypeScriptFieldTypeNameConverter as FieldTypeConverter } from "./typescriptfieldtypeconverter";
 
 export class FieldGroup implements TsExpression {
     constructor(
-        private stringFields: StringField[],
-        private calculatedFields: CalculatedField[],
-        private stringListFields: StringListField[],
-        private entityListFields: EntityListField[],
-        private fileFields: FileField[]
+        private stringFields: TSDefinedField[],
+        private calculatedFields: TSDefinedField[],
+        private stringListFields: TSDefinedField[],
+        private entityListFields: TSDefinedField[],
+        private fileFields: TSDefinedField[]
```

Comment:
I can simplify this class and fieldtype-convertert.ts, 
But If I did this modification, PR is too big...

So, I improve code with another modification.